cmd/kogito/command/install/data_index.go

```diff
@@ -74,18 +74,22 @@ func (i *installDataIndexCommand) RegisterHook() {
 	i.command = &cobra.Command{
 		Use:     "data-index [flags]",
 		Short:   "Installs the Kogito Data Index Service in the given Project",
-		Example: "data-index -p my-project --infinispan-url my-infinispan-server:11222 --kafka-url my-kafka-bootstrap:9092",
+		Example: "data-index -p my-project --kafka-url my-kafka-bootstrap:9092",
 		Long: `'install data-index' will deploy the Data Index service to enable capturing and indexing data produced by one or more Kogito Runtime Services.
 
-				kafka-url and infinispan-url are required options since the Data Index Service needs both servers to be up and running in the cluster to 
+				kafka-url is required since the Data Index Service needs this server to be up and running in the cluster to 
                 work correctly. Please refer to the https://github.com/kiegroup/kogito-cloud-operator#install-data-index-service for more information regarding
-                how to deploy an Infinispan and Kafka cluster on OpenShift for the Data Index to use.
+                how to deploy a Kafka cluster on Kubernetes for the Data Index to use.
```

Comment:
kafka environment instead cluster?

Reply:
hmm.. good eye. In fact, I used `Kafka Cluster` instead of `Kafka Environment` because of the terminology used by Strimzi. But I think that makes more sense in this context to use environment instead.

Reply:
Nevermind, should be Kafka cluster to follow up with Strimzi terminology.